common/djangoapps/pipeline_mako/__init__.py

```diff
@@ -156,3 +156,18 @@ def render_require_js_path_overrides(path_overrides):  # pylint: disable=invalid
         )
     html.append('</script>')
     return "\n".join(html)
+
+
+def require_js_path(path):
+    """
+    Returns hashed version of the `path (str)`.
+    """
+    # Calculate the full URL, including any hashes added to the filename by the pipeline.
+    # This will also include the base static URL (for example, "/static/") and the
+    # ".js" extension.
+    actual_url = staticfiles_storage.url(path)
```

Comment:
This same code appears above, lines 140-148. Please have that code call this helper method to remove the duplication.